tools/azure-sdk-tools/devtools_testutils/sanitizers.py

```diff
@@ -174,14 +148,24 @@ def add_uri_regex_sanitizer(**kwargs):
         a simple replacement operation.
     """
 
+    request_args = _get_request_args(**kwargs)
+    _send_request("UriRegexSanitizer", request_args)
+
+
+def _get_request_args(**kwargs):
+    # type: (**Any) -> Dict
+    """Returns a dictionary of sanitizer constructor headers"""
+
     request_args = {}
-    request_args["value"] = kwargs.get("value") or "fakevalue"
-    request_args["regex"] = (
-        kwargs.get("regex") or "(?<=\\/\\/)[a-z]+(?=(?:|-secondary)\\.(?:table|blob|queue)\\.core\\.windows\\.net)"
```

Comment:
why getting rid of the second regex in your updates?

Reply:
Having a storage-centric URI regex as the default regex doesn't really seem ideal going forward, so I thought I'd remove it while doing the refactoring to get request args from a helper method -- I'll try tracking down a more general URI regex that we could use here as a default, but until then I think it makes sense to remove since we have a regex that will work with Tables during the test refactoring